packages/gravity-ui-nunjucks/gulp/gravity.js

```diff
@@ -0,0 +1,69 @@
+// Copies Gravity files to build output
+const gulp = require('gulp');
+const filter = require('gulp-filter');
+const uiLibPaths = require('@buildit/gravity-ui-web/build-api.js');
+const pkgPaths = require('./paths.js');
+const { serverReload, serverRefreshCss } = require('./patternlab.js');
+
+const taskNamePrefix = 'gravity:';
+
+
+function makeCopyCssTask( allowEmpty = false ) {
```

Comment:
Space between arguments and parenthesis doesn't seem normal. This should get picked up by the linter.